Gemfile.lock

```diff
@@ -1,165 +0,0 @@
-GEM
```

Comment:
To enable travis support (testing many versions of puppet and facter) you cannot have a Gemfile.lock because travis then cannot install different versions of puppet to test against.